Assets/HoloToolkit/UX/Scripts/Keyboard.cs

```diff
@@ -253,7 +253,7 @@ protected override void Awake()
 
             // Setting the keyboardType to an undefined TouchScreenKeyboardType,
             // which prevents the MRTK keyboard from triggering the system keyboard itself.
-            InputField.keyboardType = (TouchScreenKeyboardType)(-1);
+            InputField.keyboardType = (TouchScreenKeyboardType)(int.MaxValue);
```

Comment:
MinValue?  (technically you might be able to connect a *lot* of keyboards, but you can't connect negative keyboards!)

Reply:
This was changed because, when building for UWP XAML, it tries to convert the value into a uint and throws an error. See: #2384